 meson.build       | 2 +-
 meson_options.txt | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index d074946e..7cacb2b3 100644
--- a/meson.build
+++ b/meson.build
@@ -88,7 +88,7 @@ if os == 'Linux' and libexecdir == 'libexec'
   libexecdir = 'lib'
 endif
 libexecdir = rootprefix / libexecdir
-rc_libexecdir = libexecdir / 'rc'
+rc_libexecdir = libexecdir / get_option('librcdir')
 rc_bindir = rc_libexecdir / 'bin'
 rc_sbindir = rc_libexecdir / 'sbin'
 sbindir = rootprefix / get_option('sbindir')
diff --git a/meson_options.txt b/meson_options.txt
index e4e7c4df..25d873d2 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -35,3 +35,5 @@ option('termcap', type : 'combo',
   description : 'the termcap library to use')
 option('zsh-completions', type : 'boolean',
   description : 'install zsh completions')
+option('librcdir', type : 'string', value : 'rc',
+  description : 'default location of rc libexec dir')
