 sh/functions.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sh/functions.sh b/sh/functions.sh
index 96df289..daad19d 100644
--- a/sh/functions.sh
+++ b/sh/functions.sh
@@ -7,11 +7,11 @@ if [ x$RC_GOT_FUNCTIONS = xyes -o -n "$(command -v ebegin 2>/dev/null)" ]; then
 	:

 # Then check for the presence of functions.sh
-elif [ -f /lib/gentoo/functions.sh ]; then
-	. /lib/gentoo/functions.sh
+elif [ -f /usr/lib/artix/functions.sh ]; then
+	. /usr/lib/artix/functions.sh

 else
-	echo "/lib/gentoo/functions.sh not found. Exiting"
+	echo "/usr/lib/artix/functions.sh not found. Exiting"
 	exit 1
 fi

